backend/homework/6/src/main/java/com/example/demo/service/TyreService.java

```diff
@@ -0,0 +1,35 @@
+package com.example.demo.service;
+
+import com.example.demo.model.Tyre;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Service;
+
+import java.util.Random;
+
+/**
+ * Service class responsible for generating Tyre objects.
+ */
+@Service
+public class TyreService {
+    private Random random = new Random();
+
+    @Bean
+    public Tyre generateTyre() {
+        Tyre tyre = new Tyre();
+        tyre.setBrand(generateRandomBrand());
+        tyre.setPrice(calculateTyrePrice(tyre.getBrand()));
+        return tyre;
+    }
+
+    private String generateRandomBrand() {
```

Comment:
This can be a common function in a util.
And arguments can be passed to identify tyre, speaker etc